requirements.txt

```diff
@@ -15,9 +15,11 @@ nbformat>=5.0
 notebook>=6.4.10
 numpy>=1.18.5; python_version <= "3.7"
 numpy>=1.19.5; python_version > "3.7"
+numpy>=1.23.0; python_version >= "3.10"
 packaging
 pandas>=1.1.0; python_version <= "3.8"
 pandas>=1.1.3; python_version > "3.8"
+pandas>=1.4.0; python_version >= "3.10"
```

Comment:
With this change do we need to put an upper bound on the `python_version <= "3.8"` lines?